### PR TITLE
Add the Views associated with the Models and Controllers

### PR DESCRIPTION
**Pull request by V-Blaze**
In this branch, These are the significant changes:

- Implemented the `index` and `show` methods in the `Users` and `Posts` controllers.
- Created the HTML markup for the `index` and `show` view files.
- Added minimal styling to match the provided wireframe.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,20 @@
...
+        <h3 class="user_bio_title">Bio</h3>
+        <span>
+            <%= @user.bio %>
+        </span>
+    </div>
+
+    <div class="user_details_post">
+        <%= render 'post', posts: @user.posts %>
+    </div>
+    <a href="/users/<%= @user.id %>/posts" class="user_details_see_btn">See all posts</a>
```
Comment (Kamba56) [inline app/views/users/show.html.erb]: - [ ] Please on the users show page, kindly make sure that only the three most recent posts are displayed by invoking the `@user.three_most_recent_post` method. Only when the user clicks on the `See all posts` before all posts by the user shall be displayed.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,30 @@
...
+                <a href="/users/<%= @user.id %>/posts/<%= post.id %>" class="">view post</a>
+            </div>
+            <div class="user_comments_info">
+                <% post.comments.each do |comment| %>
+                    <div class="user_comment_div">
+                        <span><%= comment.author.name  %>: </span>
+                        <span><%= comment.text  %></span>
+                    </div>
+                <%-end-%>
+            </div>
```
Comment (Kamba56) [inline app/views/posts/index.html.erb]: - [ ] Please on the posts index page, kindly make sure that only the five most recent comments are displayed by invoking the `@post.five_most_recent_comments` method. Only when the user clicks on the `view posts` before all post comments shall be displayed.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @V-Blaze, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed
- [x] Great job fixing N+1 problems

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (V-Blaze) [reply app/views/users/show.html.erb]: Changes implemented. 🙇

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (V-Blaze) [reply app/views/posts/index.html.erb]: Changes implemented. 🙇

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @V-Blaze, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**